pkg/test/skip/skip_test.go

```diff
@@ -0,0 +1,35 @@
+package skip_test
+
+import (
+	"os"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+
+	tskip "zotregistry.io/zot/pkg/test/skip"
+)
+
+// for code coverage.
+func TestSkipS3(t *testing.T) {
+	Convey("Test if S3MOCK_ENDPOINT env is uset", t, func() {
+		envName := "S3MOCK_ENDPOINT"
+		envVal := os.Getenv(envName)
+		if len(envVal) > 0 {
+			So(os.Setenv(envName, ""), ShouldBeNil)
+			defer So(os.Setenv(envName, envVal), ShouldBeNil)
```

Comment:
The defers should be before setting the new value, right?
Otherwise the variable would not be reset if line 18 errors.

I think same applies to the Dynamo test